xml/System/Environment.xml

```diff
@@ -666,7 +666,16 @@ goto :EOF
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Environment.GetEnvironmentVariable%28System.String%29> method retrieves an environment variable from the environment block of the current process only. It is equivalent to calling the <xref:System.Environment.GetEnvironmentVariable%28System.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of <xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>. The environment block of the current process includes the following environment variables:  
+
+The <xref:System.Environment.GetEnvironmentVariable%28System.String%29> method retrieves an environment variable from the environment block of the current process only. It is equivalent to calling the <xref:System.Environment.GetEnvironmentVariable%28System.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of <xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>. 
+
+To retrieve all environment variables along with their values, call the <xref:System.Environment.GetEnvironmentVariables%2A> method.  
+  
+Environment variable names are not case-sensitive.  
+ 
+### On Windows systems
+
+ On Windows systems, the environment block of the current process includes the following environment variables:  
```

Comment:
This description might be misleading as well. On both Windows and Unix, a process inherits the environment variables provided to it by the parent process that created it. 

The per-machine and per-user environment variables are used to initialize the environment block of processes with no parent (e.g. processes started from Windows Start menu - the operating system itself is the parent). The per-machine or per-user environment variable can be modified or cleared by the current process or one of its parent. The value of per-machine or per-user variable in the current process may be different from its defined value, or the current process might not have that variable set at all.